spring-cloud-stream-app-generator/pom.xml

```diff
@@ -154,6 +157,25 @@
 						</syslog-source>
 						<task-launcher-local-sink />
 						<task-launcher-yarn-sink />
+						<task-launcher-cloudfoundry-sink>
+							<forceDependencies>
+								<dependency>
+									<groupId>io.projectreactor</groupId>
+									<artifactId>reactor-core</artifactId>
+									<version>${io-projectreactor-core.version}</version>
+								</dependency>
+								<dependency>
+									<groupId>io.projectreactor.ipc</groupId>
+									<artifactId>reactor-netty</artifactId>
+									<version>${io-projectreactor-ipc.version}</version>
+								</dependency>
+								<dependency>
+									<groupId>io.netty</groupId>
+									<artifactId>netty-all</artifactId>
+									<version>${io-netty-all.version}</version>
```

Comment:
Getting this error:
`Could not find artifact io.netty:netty-all:jar:4.1.0.FINAL`.